cypress/support/commands.ts

```diff
@@ -1,37 +1,57 @@
-/// <reference types="cypress" />
-// ***********************************************
-// This example commands.ts shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
-//
-// declare global {
-//   namespace Cypress {
-//     interface Chainable {
-//       login(email: string, password: string): Chainable<void>
-//       drag(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
-//       dismiss(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
-//       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
-//     }
-//   }
-// }
\ No newline at end of file
+
+// Custom Commands for login-page
+
+Cypress.Commands.add('login', function (email, password, rememberMe) {
+  cy.visit('auth/login');
+
+  cy.get('form input#input-email').type(email, {delay: 50});
+  cy.get('form input#input-password').type(password, {delay: 50});
+
+  if (rememberMe) {
+    cy.get('form .custom-checkbox').click();
+  }
+
+  cy.get('form button[status="primary"]').click();
+
+  cy.url().should('contain', '/pages');
+});
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// Custom Commands for smart-table
+
+Cypress.Commands.add('member', function (id, firstName, lastName, username, email, age ) {
+
+  cy.visit('/pages/tables/smart-table');
+
+  cy.get('i.nb-plus').click();
+
+  cy.get('input-editor').eq(0).type(id, {delay: 50});
+  cy.get('input-editor').eq(1).type(firstName, {delay: 50});
+  cy.get('input-editor').eq(2).type(lastName, {delay: 50});
+  cy.get('input-editor').eq(3).type(username, {delay: 50});
+  cy.get('input-editor').eq(4).type(email, {delay: 50});
+  cy.get('input-editor').eq(5).type(age, {delay: 50});
+
+  cy.get('.ng2-smart-action.ng2-smart-action-add-create').click();
+
+  cy.get('table-cell-view-mode').eq(0).should('have.text', '61');
```

Comment:
Where does come values you have in assets from? you test will fail if you pass another values